llama-index-core/llama_index/core/indices/property_graph/sub_retrievers/vector.py

```diff
@@ -41,13 +41,15 @@ def __init__(
         vector_store: Optional[VectorStore] = None,
         similarity_top_k: int = 4,
         path_depth: int = 1,
+        filters: Optional[MetadataFilters] = None,
         **kwargs: Any
     ) -> None:
         self._retriever_kwargs = kwargs or {}
         self._embed_model = embed_model or Settings.embed_model
         self._similarity_top_k = similarity_top_k
         self._vector_store = vector_store
         self._path_depth = path_depth
+        self._filters = filters
```

Comment:
I think technically this was supported with line 47 no? (But its cool, it works here)